protocol/x/feetiers/keeper/keeper.go

```diff
@@ -97,6 +100,15 @@ func (k Keeper) getUserFeeTier(ctx sdk.Context, address string) (uint32, *types.
 		idx = uint32(i)
 	}
 
+	// Bump up to RefereeStartingFeeTier if the user is referred by an affiliate.
+	// We subtract 1 because the fee tiers are 1-indexed.
```

Comment:
It's a bit annoying that we have to do this, thanks for being attentive to details here.

Feel like below would be cleaner: 
- Rename constant to `RefereeStartingFeeTierIndex`, make this `2`, and add a comment this is 0-indexed and refers to the index of the feeTiers array instead of fee tier name. This way we aren't making assumption that fee tier name is well aligned to the indexes.